assignments/arrays.js

```diff
@@ -63,32 +63,51 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('Car 33 is a', inventory[32].car_year, inventory[32].car_make, inventory[32].car_model);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(inventory[inventory.length - 1].car_make, inventory[inventory.length -1].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for (let i = 0; i< inventory.length; i ++) {
+    carModels.unshift(inventory[i].car_model);
```

Comment:
interesting use of unshift. People normally use push to add the item to the end of the array, and only use unshift to add the item to the beginning of the array if they have a purpose for pushing it to the front. Nothing wrong with the way you did it. Just different then I'm used to seeing it. Good job.